lighthouse-core/audits/server-response-time.js

```diff
@@ -22,7 +22,10 @@ const UIStrings = {
 
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
-const RESPONSE_THRESHOLD = 600;
+// Due to the way that DevTools throttling works we cannot see if server response took less than ~570ms.
+// We set our failure threshold to 600ms to avoid those false positives but we want devs to shoot for 100ms.
```

Comment:
Does this mean that a passing response time will always be between 570 and 600 ms or am I understanding this wrong?

Reply:
When `throttlingMethod` is `devtools` that's right. The minimum response time we'll observe for all requests is `3.75 * 150 = 562.5ms` so a passing request will be right around there.

Reply:
Gotcha! Thanks for clearing that up for me.